general/src/pii/processor.rs

```diff
@@ -91,9 +92,12 @@ lazy_static! {
             ")([\\s]|[[:punct:]]|$)",
         )
     ).unwrap();
+
+    // http://www.richardsramblings.com/regex/credit-card-numbers/
     static ref CREDITCARD_REGEX: Regex = Regex::new(
-        r#"(?x)
-            \d{4}[- ]?\d{4,6}[- ]?\d{4,5}(?:[- ]?\d{4})
+        r#"(?x)(
+            \b(?:3[47]\d|(?:4\d|5[1-5]|65)\d{2}|6011)\d{12}\b
```

Comment:
CC numbers are often times formatted with spaces or dashes. Can we somehow bring those back? That might complicate the regex, however, as those spaces are different for Amex and VISA/MasterCard. 
Also, this gets me wondering now if there are any other newer CC companies, that would need to be added? The blog post is from 2012, so it might be outdated.

Reply:
Done. I couldn't figure out how people format the various creditcard formats so I am now allowing spaces and dashes between almost every digit.